x-pack/plugins/session_view/public/components/session_view/index.tsx

```diff
@@ -43,10 +44,13 @@ export const SessionView = ({
   const [selectedProcess, setSelectedProcess] = useState<Process | null>(null);
   const [searchQuery, setSearchQuery] = useState('');
   const [searchResults, setSearchResults] = useState<Process[] | null>(null);
-  const [displayOptions, setDisplayOptions] = useState<DisplayOptionsState>({
-    timestamp: true,
-    verboseMode: true,
-  });
+  const [displayOptions, setDisplayOptions] = useLocalStorage<DisplayOptionsState>(
+    'displayOptions',
```

Comment:
let's declare this value within a `const` under `session_view/common/constants.ts`, also let's scope the value to `sessionView:` to avoid further conflicts: i.e:

```js
// constants.ts
// ...
// local storage keys
const LOCAL_STORAGE_DISPLAY_OPTIONS_KEY = 'sessionView:displayOptions' 
```